docs/user/openstack/known-issues.md

```diff
@@ -165,3 +165,16 @@ $ rm -f openshift/99_openshift-cluster-api_worker-machineset-*.yaml
 ```
 [bugzilla-upi]: https://bugzilla.redhat.com/show_bug.cgi?id=1927244
 [manifests-removal]:../openstack/install_upi.md#remove-machines-and-machinesets
+
+## Limitations of creating external load-balancers using pre-defined FIP
+
+By default, installations with in-tree cloud provider enabled does support usage
+of a Floating IP address that was defined in the Service specification,
+since only admin users can create floating IP with specific address. It's
+possible to [change the policy to also allow non-admin users][policy-change-steps].
+If no policy change is wanted, the user should not provider a `loadBalancerIP` in
+the Service spec to create an external load-balancer with the in-tree provider.
+When using the out-of-tree cloud provider, a floating IP pre-created by the user
+can be used.
```

Comment:
I still find the phrasing a little bit confusing as it kinda suggests the policy problem only affects in-tree cloud provider.
How about something like the following?
```suggestion
On most clouds, the default policy prevents non-admin users from creating                                     
a Floating IP with a specific address. This causes the cloud provider to fail
provisioning Load Balancers where a Floating IP address is present in the
Service specification.

It's possible to pre-create a floating IP and pass it in the Service
specification when using the external cloud provider. The in-tree cloud
provider does not support this.
Alternatively, it's also possible to [relax the policy][policy-change-steps]
and allow non-admin users to create FIPs with specific IP.
```